src/activity/activity.jsx

```diff
@@ -0,0 +1,141 @@
+import { h, Component } from 'preact'
+import cx from 'classnames'
+
+import * as _ from 'utils'
```

Comment:
Is there a reason why you have a custom set of utils for lodah-like methods? If bundle size is a concern, you can import lodash modules individually.